Source/WebKit/UIProcess/Cocoa/PageClientImplCocoa.mm

```diff
@@ -435,7 +435,8 @@
 
 void PageClientImplCocoa::didCommitLayerTree(const RemoteLayerTreeTransaction& transaction)
 {
-    [webView() _updateFixedContainerEdges:transaction.fixedContainerEdges()];
+    if (transaction.fixedContainerEdges())
+        [webView() _updateFixedContainerEdges:*transaction.fixedContainerEdges()];
```

Comment:
nit:
```suggestion
    if (const auto& edges = transaction.fixedContainerEdges())
        [webView() _updateFixedContainerEdges:*edges];
```

Reply:
Fixed! I opted for just `auto&`, since we _tend_ to omit the `const` in most other places in the codebase.